project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject hikari-cp "1.7.5"
+(defproject hikari-cp "1.7.6"
```

Comment:
Thank you @sbitteker but could you, please, revert this commit? I don't want to mix two things - adding a feature and bumping a version (and therefore preparing a release) are two separate things :)